### ISSUE TITLE
request: description for gift cards

### ISSUE THREAD

**Issue by j-herden** [pretix/pretix#1558]
It would be nice if I could input a description for each gift card. 
Use case: keep a record to whom I gave the code

A plus would be, if searching in gift cards would look into this new field too.

**Comment by raphaelm** [comment]
Agreed. In case anyone wants to work on this, I'd rather call it a comment than a description since it stays internal.
